src/main/resources/resources/logisim/strings/std/std_es.properties

```diff
@@ -774,6 +774,7 @@ TTL7414 = 7414: inversor hexagonal (schmitt-trigger)
 TTL74161 = 74161: 4-bit sync counter with clear
 # ==> TTL74163 =
 TTL74165 = 74165: Registro de desplazamiento paralelo a serie de 8 bits
+# ==> TTL74166 = 74166: Registro de desplazamiento paralelo a serie de 8 bits - FIXME: "with asynchronous clear"
```

Comment:
Please remove anything but the key. So it should be just

```ini
# ==> TTL74166
```

applies to all the cases below as well. One can always use [trans-tool](https://github.com/MarcinOrlowski/trans-tool) to get reference strings during works on their translation

Reply:
existing empty translations all include the equal sign as well, i.e.
```
# ==> TTL74166 =
```

should it not be done that way?

Reply:
It's not the case for all the translation files, hence my request.